queue.h

```diff
@@ -0,0 +1,48 @@
+#ifndef CONCURRENT_QUEUE_H
+#define CONCURRENT_QUEUE_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifdef _WIN32
+#if defined(CQ_STATIC) //preferred way
+#define CQ_EXPORT
```

Comment:
All macros and symbols should be prefixed with `moodycamel`

Reply:
uppercase is ok?

Reply:
Yes, for macros